wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -18,13 +18,40 @@
             "default": "*"
           }
         },
+        {
+          "widget-type": "radio-group",
+          "name": "preconditionToggle",
+          "label": "Precondition Toggle",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "jexl",
+            "options": [
+              {
+                "id": "jexl",
+                "label": "JEXL"
+              },
+              {
+                "id": "sql",
+                "label": "SQL"
+              }
+            ]
+          },
+        },
         {
           "widget-type": "textbox",
           "label": "Precondition",
           "name": "precondition",
           "widget-attributes" : {
             "default" : "false"
           }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Precondition",
```

Comment:
We should make it clear that this textbox will accept a SQL precondition in the label. Similar for JEXL.

Reply:
I would simply name this "Precondition (SQL)".